README.md

```diff
@@ -27,8 +27,91 @@
 *Vue language client for coc.nvim*
 
 [neovim/nvim-lspconfig](https://github.com/neovim/nvim-lspconfig) ⚡ 🤝 \
-*Vue language server configuration for Neovim* \
-[[Volar 2.0 version set up tutorial](https://github.com/vuejs/language-tools/issues/3925)]
+*Vue language server configuration for Neovim*
+
+<details>
+  <summary>How to configure vue language server with neovim and lsp?</summary>
+
+## Configuration for versions:  `^2.0.0`
+
+Note: take over mode has been removed. You have to run `@vue/language-server` alongside a TypeScript server that is running `@vue/typescript-plugin`. Here is a minimal configuration for Neovim's LSP to make the language server work after upgrading to version `2.0.0`.
+
+  ```lua
+-- If you are using mason.nvim, you can get the ts_plugin_path like this
+-- local mason_registry = require('mason-registry')
+-- local ts_plugin_path = mason_registry.get_package('vue-language-server'):get_install_path() .. '/node_modules/@vue/language-server/node_modules/@vue/typescript-plugin'
+
+local ts_plugin_path = '/path/to/@vue/typescript-plugin'
+
+local lspconfig = require('lspconfig')
+
+lspconfig.tsserver.setup {
+  init_options = {
+    plugins = {
+      {
+        name = '@vue/typescript-plugin',
+        location = ts_plugin_path,
+        languages = { 'vue' },
+      },
+    },
+  },
+  filetypes = { 'typescript', 'javascript', 'javascriptreact', 'typescriptreact', 'vue' },
+}
+
+lspconfig.volar.setup {}
+
+-- Auto cmd (LspAttach) to setup keybind, codelens, and formatting stuff
+-- I assume everyone should have this configured already but just for reference
+-- @see https://github.com/nvim-lua/kickstart.nvim/blob/65a5ac404b56c4718d79f65ac642e19e89346eda/init.lua#L451-L522
+  ```
+### Hybrid mode(similar to takeover mode) configuration (Requires `@vue/language-server` version `^2.0.7`)
```

Comment:
Here says the opposite:
Disabling Hybrid Mode means that TS/CSS/HTML language support in .vue files is provided by @vue/language-server.
Enabling Hybrid Mode means that TS support within .vue files is provided by tsserver + @vue/typescript-plugin, CSS/HTML support is still provided by @vue/language-server,
If you want the behavior of v1 takeover mode, you actually need to disable Hybrid Mode.

Reply:
Yeah I realized that, my brain does not work properly atm